pkg/user/user.go

```diff
@@ -33,17 +33,13 @@ type uidParser struct {
 	placeholder string
 	uid         int
 	gid         int
-	finger      []string
-	userdir     string
-	shell       string
```

Comment:
please help to check whether these fields are useful in your scope. @Ace-Tang

Reply:
The code have been changed by @rudyfly once, I think it is why make these filed unused, but the function should be fine